frontend/src/styles/documentlist.scss

```diff
@@ -33,11 +33,41 @@
   margin: 0;
 }
 
-Button {
+.expandButton {
   margin: 0;
   padding: 0;
+  border-color: #ffffff;
+  margin-bottom: 1.5px;
+  width: 100%;
 }
 
+
 .modal-footer Button {
   margin-right: 1px;
+}
+
+.expandButtonImage{
+  max-width: 25px;
+  max-height: 25px;
+  border: none;
+}
+
+#buttonRowData {
+  padding: 1%;
+}
+
+.ApprovedColor {
```

Comment:
I believe these colors should be included in the index.scss file, so you might be able to import index.css and then use those classnames.